lib/openhab/core/types/date_time_type.rb

```diff
@@ -51,6 +51,17 @@ def to_zoned_date_time(context = nil) # rubocop:disable Lint/UnusedMethodArgumen
           zoned_date_time
         end
 
+        # @!visibility private
+        def to_instant(context = nil) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
just rename this to `_context = nil`, then you won't have to manually disable the cop